app/src/main/java/io/intrepid/contest/screens/contestjudging/submissionentrieslist/SubmissionEntriesActivity.java

```diff
@@ -0,0 +1,62 @@
+package io.intrepid.contest.screens.contestjudging.submissionentrieslist;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.BindView;
+import io.intrepid.contest.R;
+import io.intrepid.contest.base.BaseMvpActivity;
+import io.intrepid.contest.base.PresenterConfiguration;
+import io.intrepid.contest.models.Entry;
+import io.intrepid.contest.screens.contestjudging.EntryListAdapter;
+
+public class SubmissionEntriesActivity extends BaseMvpActivity<SubmissionEntriesPresenter>
```

Comment:
This name is kind of confusing, since we have the package `entrysubmission` for the actual submission by the contestant.
Maybe you can name it `scoresubmission` or something of the sort.